src/app/lib/utilities/getEmbedUrl/index.js

```diff
@@ -2,8 +2,10 @@ import isLive from '../isLive';
 
 const AV_ROUTE = 'ws/av-embeds';
 
-const LIVE_URL = 'https://polling.bbc.co.uk';
-const TEST_URL = 'https://polling.test.bbc.co.uk';
+const LIVE_CANONICAL_URL = 'https://bbc.co.uk';
```

Comment:
Worth trying .com instead of .co.uk?

Reply:
![022](https://user-images.githubusercontent.com/9042672/90798235-95767280-e309-11ea-8582-c7356fbf5853.jpg)

Reply:
Sorry I just _had_ to use that image. 

.com makes a lot more sense - I was just mimicking what we already had. Do you know why we had .co.uk here previously?

Reply:
It was a request from James, when I did this first time round. Prevents an unnecessary redirect?

Reply:
@jamesdonoh do you recall https://github.com/bbc/simorgh/pull/5493#discussion_r381363721?

Reply:
We should use .com - WS users will be on bbc.com - if we use a .co.uk domain for the embed endpoint, that will be another DNS lookup - something this PR was intended to solve

It also seems to be the case that .co.uk redirects to .com anyway on the /ws subpath: https://www.test.bbc.co.uk/ws/av-embeds/legacy/persian/iran/2016/09/160907_tc2_testmap1/123/ru

Reply:
Hey just to clarify for future code explorers - the `.com` change only works on canonical (i.e. without the `polling.` domain). 

For AMP, with `polling` - we still need to use .co.uk